oci/table_oci_identity_tag_namespace.go

```diff
@@ -0,0 +1,231 @@
+package oci
+
+import (
+	"context"
+	"strings"
+
+	oci_common "github.com/oracle/oci-go-sdk/v36/common"
+	"github.com/oracle/oci-go-sdk/v36/identity"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableIdentityTagNamespace(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "oci_identity_tag_namespace",
+		Description: "OCI Identity Tag Namespace",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("id"),
+			Hydrate:    getIdentityTagNamespace,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listIdentityTagNamespaces,
+		},
+		GetMatrixItem: BuildCompartmentList,
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the tag namespace. It must be unique across all tag namespaces in the tenancy and cannot be changed.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "The OCID of the tag namespace.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "is_retired",
+				Description: "Whether the tag namespace is retired.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "lifecycle_state",
+				Description: "The tagnamespace's current state.",
+				Type:        proto.ColumnType_STRING,
+			},
+
+			// Other columns
+			{
+				Name:        "description",
+				Description: "The description you assign to the tag namespace.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "Date and time the tagNamespace was created",
```

Comment:
```suggestion
				Description: "Date and time the tagNamespace was created.",
```
`time_created` is important column, it should be above `description`.